### PR TITLE
GPII-3882: New button - launch DocuMorph.exe

### PR DESCRIPTION
**Pull request by Karadaliev**
* Added new type of service button for the executing a file
* Added new service button for the Launch DocuMorph to QSS

### CODE REVIEW

**Comment by Karadaliev** [comment]
@javihernandez Can you send me the proper path to the DocuMorph executable when you know where it will be installed? Because right now for testing purposes I'm executing the path to Notepad ;-).

**Comment by javihernandez** [comment]
ok to test

**Comment by Karadaliev** [comment]
@javihernandez can you run the tests once more?

**Comment by javihernandez** [comment]
ok to test

**Comment by javihernandez** [comment]
@Karadaliev can you please resolve conflicts with master?

File: src/main/common/utils.js [hunk 1]
```diff
@@ -196,10 +197,31 @@ gpii.app.isPointInRect = function (point, rectangle) {
...
+
+/**
+ * A custom function for handling opening of an .exe file.
+ * @param {String} executablePath - path to executable file
+ */
+gpii.app.launchExecutable = function (executablePath) {
+    var fileProperties = fs.statSync(executablePath);
+
+    // Check that the file is executable
+    if (fileProperties.mode === 33206) {
```
Comment (amb26) [inline src/main/common/utils.js]: Please use octal literal here so it is easier to understand what the required file mode is

File: src/renderer/qss/js/qssServiceButtons.js [hunk 2]
```diff
@@ -206,6 +206,29 @@
         openUSB.fire();
     };
 
+    /**
+     * Inherits from `gpii.qss.buttonPresenter` and handles interactions with the "launch .exe file"
+     * QSS button.
+     */
+    fluid.defaults("gpii.qss.launchExePresenter", {
```
Comment (amb26) [inline src/renderer/qss/js/qssServiceButtons.js]: This default is named generically, but it is hard-wired to only launch the docuMorphExecutable option from the site config. Either find a way to express what it launches parametrically (e.g. by having the particular instance of it supplying an option "docuMorphExecutable" or else the resolved siteConfig option), or rename it

**Comment by Karadaliev** [comment]
Code review changes are applied. Everything is greatly simplified, and the file mode is represented in octal literal.

File: src/main/common/utils.js (hunk 1 above)
Comment (Karadaliev) [reply src/main/common/utils.js]: Done

File: src/renderer/qss/js/qssServiceButtons.js (hunk 2 above)
Comment (Karadaliev) [reply src/renderer/qss/js/qssServiceButtons.js]: The presenter is renamed properly, and now its using a universal function to execute the file.

**Pull request merged**